helm/scalable-integrator/README.md

```diff
@@ -0,0 +1,97 @@
+# Helm Charts for deployment of Integrator profile of WSO2 Enterprise Integrator
+
+## Prerequisites
+
+* In order to use these Kubernetes resources, you will need an active [Free Trial Subscription](https://wso2.com/free-trial-subscription)
+from WSO2 since the referring Docker images hosted at docker.wso2.com contains the latest updates and fixes for WSO2 Enterprise Integrator.
+You can sign up for a Free Trial Subscription [here](https://wso2.com/free-trial-subscription).<br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). This can
```

Comment:
@KavinduZoysa IMO it is better to leave this as `Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/)`.